packages/manager/apps/pci-vouchers/src/components/vouchers/listing/DisplayName.spec.tsx

```diff
@@ -1,72 +1,68 @@
 import { describe, expect, vi } from 'vitest';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
-import { render, screen, waitFor } from '@testing-library/react';
+import { render, waitFor } from '@testing-library/react';
 import '@testing-library/jest-dom';
 import { ReactNode } from 'react';
 import DisplayName from '@/components/vouchers/listing/DisplayName';
 
 vi.mock('react-i18next', () => ({
   // this mock makes sure any components using the translate hook can use it without a warning being shown
-  useTranslation: () => {
-    return {
-      t: (str: string) => str,
-      i18n: {
-        changeLanguage: () => new Promise(() => {}),
-      },
-    };
-  },
+  useTranslation: () => ({
+    t: (str: string) => str,
+    i18n: {
+      changeLanguage: () => new Promise(() => {}),
+    },
+  }),
```

Comment:
already defined in packages/manager/apps/pci-vouchers/src/setupTests.ts so no need to define it one more time

Reply:
done